carbon-design-system/carbon-addons-iot-react
# chore(repo): v3 pr action fixes

Closes #2969 

**Summary**

- the github action to automatically create v3 prs is failing. This attempts to fix that.

**Change List (commits, features, bugs, etc)**

- update the github action with the same flow we use locally.

**Acceptance Test (how to verify the PR)**

- this can be tested in your own fork of this repo by
- add this code to `next` in your fork
- update some other piece of code in `next`
- push these changes directly to `next` in your fork
- when the actions run it should open a PR containing your changes (see https://github.com/kevinsperrine/carbon-addons-iot-react/pull/1)

**Regression Test (how to make sure this PR doesn't break old functionality)**

- n/a



**Things to look for during review**
